src/main/java/core/basesyntax/Drawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawing {
+    void figureInfo();
```

Comment:
```suggestion
    void getFigureInfo();
```

лучше начинать названия методов с глагола